Windows 10 x64 ImportError: DLL load failed - location issue or binary compatibility issue?

System info:
- using Python 3.7.2 official installation
- have VS2019 build tools (cl.exe prints `Microsoft (R) C/C++ Optimizing Compiler Version 19.21.27702.2 for x86`)

Steps taken:
- used the latest master branch of cyipopt (0.2.0-dev0), and downloaded the linked Ipopt binary versions
- python setup.py install runs fine `...Finished processing dependencies for ipopt==0.2.0.dev0`
- however, `import ipopt` prints 
```
>python
Python 3.7.2 (tags/v3.7.2:9a3ffc0492, Dec 23 2018, 23:09:28) [MSC v.1916 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> import ipopt
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Users\...\AppData\Local\Programs\Python\Python37\lib\site-packages\ipopt-0.2.0.dev0-py3.7-win-amd64.egg\ipopt\__init__.py", line 43, in <module>
    from cyipopt import *
ImportError: DLL load failed: The specified module could not be found.
```

- some DLLs are in this location
```
>ls C:\Users\...\AppData\Local\Programs\Python\Python37\Lib\site-packages\ipopt-0.2.0.dev0-py3.7-win-amd64.egg
EGG-INFO        IpOptFSS.dll   __pycache__                 cyipopt.py  msvcp100.dll
IpOpt-vc10.dll  Ipopt-vc8.dll  cyipopt.cp37-win_amd64.pyd  ipopt       msvcr100.dll
```

Debugging so far:
- I have so far looked through https://github.com/matthias-k/cyipopt/issues/21 as well as searched through all the issues in this repo for DLL and looked through the solutions.
- Is this due to a mismatch in the C compiler that produced the Ipopt binaries and the one that compiled cyipopt on my system? If so, is there a way to know which compiler was used for that, or any link to instructions for recompiling Ipopt?
- Potentially #41 would alleviate this problem

Here is the relevant call to the compiler on appveyor:

```
:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\BIN\x86_amd64\cl.exe /c /nologo /Ox /W3 /GL /DNDEBUG /MD -IC:\projects\cyipopt\include\coin -IC:\Miniconda37-x64\lib\site-packages\numpy\core\include -IC:\Miniconda37-x64\include -IC:\Miniconda37-x64\include "-IC:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\INCLUDE" "-IC:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\ATLMFC\INCLUDE" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.18362.0\ucrt" "-IC:\Program Files (x86)\Windows Kits\NETFXSDK\4.6.1\include\um" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.18362.0\shared" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.18362.0\um" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.18362.0\winrt" /Tcsrc\cyipopt.c /Fobuild\temp.win-amd64-3.7\Release\src\cyipopt.obj
cyipopt.c
c:\miniconda37-x64\lib\site-packages\numpy\core\include\numpy\npy_1_7_deprecated_api.h(14) : Warning Msg: Using deprecated NumPy API, disable it with #define NPY_NO_DEPRECATED_API NPY_1_7_API_VERSION
src\cyipopt.c(3887): warning C4244: '=': conversion from 'Py_ssize_t' to 'Index', possible loss of data
src\cyipopt.c(3945): warning C4244: '=': conversion from 'Py_ssize_t' to 'Index', possible loss of data
C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\BIN\x86_amd64\link.exe /nologo /INCREMENTAL:NO /LTCG /DLL /MANIFEST:EMBED,ID=2 /MANIFESTUAC:NO /LIBPATH:C:\projects\cyipopt\lib\x64\ReleaseMKL /LIBPATH:C:\Miniconda37-x64\libs /LIBPATH:C:\Miniconda37-x64\PCbuild\amd64 "/LIBPATH:C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\LIB\amd64" "/LIBPATH:C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\ATLMFC\LIB\amd64" "/LIBPATH:C:\Program Files (x86)\Windows Kits\10\lib\10.0.18362.0\ucrt\x64" "/LIBPATH:C:\Program Files (x86)\Windows Kits\NETFXSDK\4.6.1\lib\um\x64" "/LIBPATH:C:\Program Files (x86)\Windows Kits\10\lib\10.0.18362.0\um\x64" Ipopt-vc8.lib IpOptFSS.lib IpOpt-vc10.lib /EXPORT:PyInit_cyipopt build\temp.win-amd64-3.7\Release\src\cyipopt.obj /OUT:build\lib.win-amd64-3.7\cyipopt.cp37-win_amd64.pyd /IMPLIB:build\temp.win-amd64-3.7\Release\src\cyipopt.cp37-win_amd64.lib
cyipopt.obj : warning LNK4197: export 'PyInit_cyipopt' specified multiple times; using first specification
   Creating library build\temp.win-amd64-3.7\Release\src\cyipopt.cp37-win_amd64.lib and object build\temp.win-amd64-3.7\Release\src\cyipopt.cp37-win_amd64.exp
```

--- a/README.rst
+++ b/README.rst
@@ -108,13 +108,13 @@ Then, execute::
 
 From source on Windows
 ~~~~~~~~~~~~~~~~~~~~~~
+ 
+Install the dependencies with conda (Anaconda or Miniconda)::
 
-Install the dependencies with conda::
-
-   conda install numpy cython future six setuptools mkl
+   conda.exe install -c conda-forge numpy cython future six setuptools mkl
 
 Additionally, make sure you have a C compiler setup to compile Python C
-extensions, e.g.  Visual C++.
+extensions, e.g.  Visual C++. Build tools for VS2019 https://visualstudio.microsoft.com/downloads/#build-tools-for-visual-studio-2019 have been tested to work for this (see https://github.com/matthias-k/cyipopt/issues/52).
 
 Download and extract the cyipopt source code from Github or PyPi.
 
@@ -131,6 +131,9 @@ the directory that contains the ``lib`` and ``include`` directories.
 Finally, execute::
 
    python setup.py install
+
+**NOTE:** 
+Only conda Python is supported at the moment, and *not* the official python.org distribution. There is likely a binary compatibility issue between the C compiler used to build IPOPT and that used for compiling Python extensions in the official Python distribution (see https://github.com/matthias-k/cyipopt/issues/52).
 
 Example Installation on Ubuntu 18.04 Using Dependencies Installed Via APT
 -------------------------------------------------------------------------

--- a/README.rst
+++ b/README.rst
@@ -114,7 +114,7 @@ Install the dependencies with conda (Anaconda or Miniconda)::
    conda.exe install -c conda-forge numpy cython future six setuptools mkl
 
 Additionally, make sure you have a C compiler setup to compile Python C
-extensions, e.g.  Visual C++. Build tools for VS2019 https://visualstudio.microsoft.com/downloads/#build-tools-for-visual-studio-2019 have been tested to work for this (see https://github.com/matthias-k/cyipopt/issues/52).
+extensions, e.g.  Visual C++. Build tools for VS2019 https://visualstudio.microsoft.com/downloads/#build-tools-for-visual-studio-2019 have been tested to work for conda Python 3.7 (see https://github.com/matthias-k/cyipopt/issues/52).
 
 Download and extract the cyipopt source code from Github or PyPi.
 
@@ -133,7 +133,7 @@ Finally, execute::
    python setup.py install
 
 **NOTE:** 
-Only conda Python is supported at the moment, and *not* the official python.org distribution. There is likely a binary compatibility issue between the C compiler used to build IPOPT and that used for compiling Python extensions in the official Python distribution (see https://github.com/matthias-k/cyipopt/issues/52).
+Only conda Python has been tested to work at the moment, and *not* the official python.org distribution. There is likely a binary compatibility issue between the C compiler used to build IPOPT and that used for compiling Python extensions in the official Python distribution (see https://github.com/matthias-k/cyipopt/issues/52).
 
 Example Installation on Ubuntu 18.04 Using Dependencies Installed Via APT
 -------------------------------------------------------------------------

